docs/_build/html/_sources/announcement.rst.txt

```diff
@@ -1,2 +1,31 @@
+.. role:: python(code)
+    :language: python
+
 Announcement Page
-==================
\ No newline at end of file
+==================
+
+This page consists of the announcements about upcoming events and recent events by your Girlscript Chapter.
+It consists of 2 sections which are as follows:
+
+1. **Upcoming events**: This section consists of the upcoming events in a timeline format.
+
+.. image:: ./images/announcementpage/upcomingevent.JPG
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+If you want to make changes in this section, carry out the following changes in code:
+
+- To make changes in the heading - Go to `announcement.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/announcement.html>`__ and search :python:`recent`, you will see <h2> tag where you can make changes.
+
+* To make changes in the styling of heading - Go to `announcement.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/announcement.css>`__ and search :python:`.recent` and do the desired changes.
```

Comment:
We don't want the user to bother about changing the style but rather the content. Hence this is not required. (As we have already taken care of that). Make similar reductions in the home page wherever necessary